src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -30,15 +30,15 @@ private class FindLiteralsProgressAdapter : IStreamingFindLiteralReferencesProgr
                 _definition = definition;
             }
 
-            public async Task OnReferenceFoundAsync(Document document, TextSpan span)
+            public async Task OnReferenceFoundAsync(Document document, TextSpan span, CancellationToken cancellationToken)
             {
                 var documentSpan = await ClassifiedSpansAndHighlightSpan.GetClassifiedDocumentSpanAsync(
                     document, span, _context.CancellationToken).ConfigureAwait(false);
```

Comment:
so I guess what you are saying is engine itself might still use that but user service will use one from StreamJson?